README.md

```diff
@@ -1 +1,50 @@
 # Apache Accumulo Testing Suite
+
+The Apache Accumulo testing suite contains applications that test and verify the
+correctness of Accumulo.
+
+## Installation
+
+In order to run the Apache Accumulo testing suite, you will need Java 8 and Maven installed
+on your machine as well as an Accumulo instance to use for testing.
+
+1. First clone this repository.
+
+        git clone git@github.com:apache/accumulo-testing.git
+        cd accumulo-testing
+
+2. All configuation files for the test suite are in `conf/`. Only the `accumulo-testing.props`
+   configuration file needs to be created and edited. All other configuration files are optional.
+
+        cd conf/
+        cp accumulo-testing.props.example accumulo-testing.props
+        vim accumulo-testing.props
+
+3. Tests are run using the `accumulo-testing` command which is located in the `bin/`
+   directory. Run this command without any arguments to view its usage and see available tests.
+
+        ./bin/accumulo-testing
+
+## Random walk test
+
+The random walk test generates client behavior on an Apache Accumulo instance by randomly walking a
+graph of client operations. The test is configured by the `accumulo-testing.props` file and requires
```

Comment:
Define here what the minimum set of properties are? Also, would prefer the standard "properties" suffix, but that's just me being picky.